deepmd/pt/utils/stat.py

```
@@ -109,50 +112,32 @@ def make_stat_input(
 
 
 def _restore_from_file(
-    stat_file_path: DPPath,
+    stat_file_path: DPPath | None,
     keys: list[str] = ["energy"],
```

Ruff (0.15.21)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)